### PR TITLE
weather: init at 2.0

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Kinda hacky approach for copying the things to the store, but as this is shipped without a makefile, setup.py or whatever, I guess I have to do it this way, right?

Works locally, though I didn't do extensive testing.

### CODE REVIEW

**Comment by jagajaga** [comment]
cc @domenkozar

**Comment by domenkozar** [comment]
If it doesn't ship with `setup.py`, there is no reason to use `buildPythonPackage`. Just use `mkDerivation`

File: pkgs/applications/misc/weather/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pythonPackages.buildPythonPackage rec {
+    version = "2.0";
+    name = "weather-${version}";
+
+    src = fetchurl {
+        url = "http://fungi.yuggoth.org/weather/src/${name}.tar.xz";
+        sha256 = "0yil363y9iyr4mkd7xxq0p2260wh50f9i5p0map83k9i5l0gyyl0";
+    };
+
+    doTest = false;
```
Comment (domenkozar) [inline pkgs/applications/misc/weather/default.nix]: `doTest` doesn't exist

**Comment by matthiasbeyer** [comment]
Doesn't `buildPythonPackage` help me with patching the `#!`, for example?

**Comment by domenkozar** [comment]
No, `wrapPython` does.

**Comment by domenkozar** [comment]
@matthiasbeyer did you test the package? I see no wrappers so I'm not sure this would work.

**Comment by matthiasbeyer** [comment]
I test-called it as `./result/bin/weather -h` and it worked. I wouldn't merge, though, I want to do some more testing asap.

**Comment by matthiasbeyer** [comment]
Why did you merge?

**Pull request merged**